test/release/examples/primers/domains.chpl

```diff
@@ -30,49 +30,46 @@
 config var n = 10;
 
 var RD: domain(3) = {1..n, 1..n, 1..n};
-writeln(RD);
+writeln(RD); // {1..10, 1..10, 1..10}
 
 //
 // Rectangular domains have a set of methods that enable convenient
 // reuse of existing domains.
 //
 // The :proc:`~ChapelDomain.expand` method returns a new domain that is expanded or
-// contracted depending on the sign of the offset argument.
+// contracted depending on the sign of the offset argument.  The
+// expansion/contraction happens symmetrically to the lower and upper bound.
```

Comment:
Please consider adding "in each dimension" somewhere in the above.

Reply:
Good call, done in latest commit with reset of requested changes.